src/hooks/http.js

```diff
@@ -78,25 +79,32 @@ export const wrappedHttpResponseCallback = (
 ) => response => {
   const clonedResponse = cloneResponse(response);
   const clonedResponsePassThrough = cloneResponse(response);
-  const { headers, statusCode } = clonedResponse;
-  const receivedTime = new Date().getTime();
-
-  let body = '';
-  clonedResponse.on('data', chunk => (body += chunk));
-
-  let responseData = {};
-  clonedResponse.on('end', () => {
-    responseData = {
-      statusCode,
-      receivedTime,
-      body,
-      headers: lowerCaseObjectKeys(headers),
-    };
-    const fixedRequestData = noCirculars(requestData);
-    const fixedResponseData = noCirculars(responseData);
-    const httpSpan = getHttpSpan(fixedRequestData, fixedResponseData);
-    SpansContainer.addSpan(httpSpan);
-  });
+  try {
```

Comment:
Why you don't use here safeExcute ?

Reply:
`safeExecute` works only on functions, I can't use it on specific like context-manager.
I don't want to use `safeExecute` on the entire function because I don't want to catch user's errors.